src/Places/index.ts

```diff
@@ -96,3 +99,79 @@ export async function detectParentsByBounds(bounds: Bounds, zoom: number): Promi
 export async function detectParentsByLocation(location: Location): Promise<Place[]> {
 	return Dao.detectParentsByLocation(location);
 }
+
+export async function getPlacesMapFromTrip(trip: Trip): Promise<Map<string, Place>> {
+	const placesMapFromTrip: Map<string, Place> = new Map<string, Place>();
+
+	if (!trip.days) {
+		return placesMapFromTrip;
```

Comment:
throw error otherwise you can't recognize empty trip from developer mistake when developer forgot to preload trip